source/index.html

```diff
@@ -1,12 +1,207 @@
+      </section>
+
+      <section class="contacts">
+        <h2 class="contacts__title">Контакты</h2>
+
+        <dl class="contacts__list contacts-list">
+          <dt class="contacts-list__item">e-mail:</dt>
+          <dd class="contacts-list__description"><a href="mailto:info@mimimishkashop.ru">info@mimimishkashop.ru</a></dd>
+          <dt class="contacts-list__item">адрес:</dt>
+          <dd class="contacts-list__description">г. Санкт-Петербург<br>ул. Большая Конюшенная<br>д. 19/8, офис 101</dd>
+        </dl>
+
+        <img class="contacts__map" src="img/map-mobile.jpg" width="320" height="454" alt="Карта местонахождения.">
+        <a class="contacts__link button button--ghost" href="#">Нaпишите нам</a>
+      </section>
+    </main>
+
+    <footer class="page-footer">
+      <a class="page-footer__logo" href="#">
+        <img class="page-footer__logo-image" src="img/logo-footer.svg" width="102" height="34" alt="Мишка.">
+      </a>
+
+      <ul class="page-footer__list socials-list">
+        <li class="socials-list__item">
+          <a class="socials-list__link" href="https://www.instagram.com/htmlacademy/">
+            <img class="socials-list__icon" src="img/icon-instagram.svg" width="22" height="21" alt="instagram.">
+          </a>
+        </li>
+        <li class="socials-list__item">
+          <a class="socials-list__link" href="https://www.facebook.com/htmlacademy">
+            <img class="socials-list__icon" src="img/icon-facebook.svg" width="15" height="28" alt="facebook.">
+          </a>
+        </li>
+        <li class="socials-list__item">
+          <a class="socials-list__link" href="https://twitter.com/htmlacademy_ru">
+            <img class="socials-list__icon" src="img/icon-twitter.svg" width="23" height="20" alt="twitter.">
+          </a>
+        </li>
+      </ul>
+
+      <div class="page-footer__copyright copyright">
+        <b class="copyright__text">Разработано</b>
+        <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">
+          <img class="copyright__image" src="img/icon-htmlacademy.svg" width="27" height="34" alt="HTML Academy.">
+        </a>
+      </div>
+    </footer>
+
+    <div class="page-modal">
+      <h2 class="page-modal__title">Добавить <br>в корзину</h2>
+      <p class="page-modal__text">Выберите размер:</p>
+      <form class="page-modal__form" action="https://echo.htmlacademy.ru" method="POST">
+        <ul class="page-modal__list modal-list">
+          <li class="modal-list__item">
+            <label class="modal-list__label" for="modal-size-small"><span class="visually-hidden">Размер: </span>S</label>
+            <input class="visually-hidden modal-list__input" type="radio" name="modal-size" id="modal-size-small" value="modal-size-small" checked>
+          </li>
+          <li class="modal-list__item">
+            <label class="modal-list__label" for="modal-size-medium"><span class="visually-hidden">Размер: </span>M</label>
```

Comment:
Так было бы читаемее, кажется
```suggestion
            <label class="modal-list__label" for="modal-size-medium">
              <span class="visually-hidden">Размер: </span>
              M
            </label>
```

Reply:
нет спана - нет проблем